.github/workflows/pylint.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10"]
```

Comment:
Could be a reason for the pylint spitting out an error.

Reply:
What exactly do you mean? the python 3.10?

Reply:
There was a pylint error, comment above points out that this _could be_ the source of the error. Because python 3.8, 3.9 and 3.10 are replaced by python 3.8
Since the pylint error is not present any more, this thread can be resolved